packages/web-react/src/components/UNSTABLE_Header/README.md

```diff
@@ -0,0 +1,177 @@
+# UNSTABLE Header
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The `UNSTABLE_Header` component is planned to replace the `Header` component in the future.
+
+This component provides these components:
```

Comment:
> component provides (…) components

This sounds like a recursion, doesn't it 🙂. Elsewhere (`Header`, `Modal`), we have it like this:

> Modal is a composition of several subcomponents:

Reply:
It's based on web documentation for UNSTABLE_Header [packages/web/src/scss/components/UNSTABLE_Header/README.md](https://github.com/lmc-eu/spirit-design-system/pull/1808/files#diff-2d3816df5d0800020420689f68dddb3ff3e02a20ba6876321368810acb255a68)

So we need to discuss it @crishpeen

Reply:
changed to 
```
The `UNSTABLE_Header` is a composition of several subcomponents:
```
in web

Reply:
Also changed. Thanks